service/matching/pri_matcher.go

```diff
@@ -440,10 +440,16 @@ func (tm *priTaskMatcher) emitDispatchLatency(task *internalTask, forwarded bool
 		return // should not happen but for safety
 	}
 
+	priStr := ""
+	if pri := task.getPriority().GetPriorityKey(); pri > 0 {
```

Comment:
is it intentional to use "" for priority 0?

Reply:
yes.. do you think it should be 0? I feel like 0 is misleading since it's not "priority 0", it's "use the default priority"